OpenSUSE ffmpeg-devel

gmscs opened the pull request:
Compilation of qTox under OpenSUSE requires ffmpeg-devel, not ffmpeg.

Commit:
OpenSUSE ffmpeg-devel

Compilation of qTox under OpenSUSE requires ffmpeg-devel, not ffmpeg.

agilob commented:
I stoppeding using OS some time ago, but don't you need both packages? ffmpeg and ffmpeg-devel?

Commit:
libffmpeg-devel

"'ffmpeg-devel' not found in package names. Trying capabilities.
'libffmpeg-devel' providing 'ffmpeg-devel' is already installed."

woops, sorry

gmscs commented:
it's working fine on my end without ffmpeg, i'll play around with it to make sure

[EDIT] Everything seems to be working without ffmpeg
